src/Backend/Modules/Blog/Layout/Templates/Edit.html.twig

```diff
@@ -213,8 +213,8 @@
           {{ macro.buttonIcon(geturl('Index'), 'times', 'lbl.Cancel'|trans|ucfirst, 'btn-default') }}
         </div>
         <div class="btn-group pull-right" role="group">
-          {{ macro.buttonIcon('', 'file-o', 'lbl.SaveDraft'|trans|ucfirst, 'btn-default', {"id":"saveAsDraft", "type":"submit"}) }}
-          {{ macro.buttonIcon('', 'check', 'lbl.Publish'|trans|ucfirst, 'btn-primary', {"id":"editButton", "type":"submit", "name":"edit"}) }}
+          {{ macro.buttonIcon('', 'file-o', 'lbl.SaveDraft'|trans|ucfirst, 'btn-default', {"type":"submit", "name":"saveAsDraft"}) }}
+          {{ macro.buttonIcon('', 'check', 'lbl.Publish'|trans|ucfirst, 'btn-primary', {"type":"submit", "name":"edit"}) }}
```

Comment:
`name` should be `publish` and a translation instead of hardcoded text.

Reply:
I've altered the name to publish but using a translation would jeopardize the workings of the PHP bit since it can no longer rely on consistent data.